src/tests/Interop/PInvoke/Generics/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ elseif(CLR_CMAKE_TARGET_ARCH_ARM)
 elseif(CLR_CMAKE_TARGET_ARCH_ARM64)
   add_definitions(-DTARGET_ARM64)
   add_definitions(-DTARGET_ARMARCH)
+elseif(CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
+  add_definitions(-DTARGET_LOONGARCH64)
```

Comment:
If `CLR_CMAKE_TARGET_ARCH_LOONGARCH64` is true, `-DTARGET_LOONGARCH64` is already defined. All these definitions (except for `TARGET_XARCH` and `TARGET_ARMARCH`) were added in top-level `configurecompiler.cmake`,

You can delete these redundant definitions from this block along with the one in `src/tests/Interop/PInvoke/Int128/CMakeLists.txt`. I added other missing loongarch64 bits under src/tests with riscv64 configurations: https://github.com/dotnet/runtime/commit/56095347ca1bacbc7a95e468f03f02ea808b9d9b#diff-a81209118d9d6cee9b59864790c59e3d75b15747a37ab6c07fd3dc87ce7494fa

Reply:
OK, Thanks

Reply:
> If `CLR_CMAKE_TARGET_ARCH_LOONGARCH64` is true, `-DTARGET_LOONGARCH64` is already available in `CFLAGS` and `CXXFLAGS`.

@am11 
I only find the defination `-DHOST_LOONGARCH64` within the top-level `configurecompiler.cmake` while not the `-DTARGET_LOONGARCH64`.

```
CXX_DEFINES = -DBUILDENV_DEBUG=1 -DDEBUG -DDISABLE_CONTRACTS -DGenericsNative_EXPORTS -DHOST_64BIT -DHOST_LOONGARCH64 -DHOST_UNIX -DTARGET_64BIT -DTARGET_LINUX -DTARGET_LOONGARCH64 -DTARGET_UNIX -DURTBLDENV_FRIENDLY=Debug -D_DBG -D_DEBUG -D_FILE_OFFSET_BITS=64
```